op-dispute-mon/mon/service.go

```diff
@@ -48,18 +64,31 @@ func (s *Service) initFromConfig(ctx context.Context, cfg *config.Config) error
 	if err := s.initL1Client(ctx, cfg); err != nil {
 		return fmt.Errorf("failed to init l1 client: %w", err)
 	}
+	if err := s.initPollClient(ctx, cfg); err != nil {
+		return fmt.Errorf("failed to init poll client: %w", err)
+	}
```

Comment:
We shouldn't need the poll client here. We're doing everything based off a time ticker, not based off the head block advancing.